website/pages/guides/integrations/with-hook-form.mdx

```diff
@@ -6,6 +6,82 @@ author: abheist
 ---
 
 This example shows how to build a simple form with Chakra UI's form components
-and the [React Hook Form](https://react-hook-form.com) form library.
+and the [React Hook Form](https://react-hook-form.com) form library:
+
+```
```

Comment:
This change is needed for syntax highlighting to work and also ensures it's not editable.

```suggestion
```jsx live=false
```